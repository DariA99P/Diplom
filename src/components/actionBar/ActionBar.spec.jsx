import React from 'react';
import { shallow } from 'enzyme';

import ActionBar from '.';

describe('<ActionBar />', () => {
  let tree;
  beforeEach(() => {
    tree = shallow(<ActionBar />);
  });

  describe('On rendering', () => {
    it('should render properly', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
