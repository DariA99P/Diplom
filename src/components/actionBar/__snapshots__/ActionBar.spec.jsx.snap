// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ActionBar /> On rendering should render properly 1`] = `
<Row
  className="actionBar"
  gutter={0}
>
  <Col
    span={8}
    style={
      Object {
        "alignItems": "flex-end",
        "flexWrap": "wrap",
        "justifyContent": "flex-start",
        "position": "flex",
      }
    }
  >
    <TabWrapper />
  </Col>
  <Col
    span={8}
    style={
      Object {
        "alignItems": "center",
        "flexWrap": "wrap",
        "justifyContent": "center",
        "position": "flex",
      }
    }
  >
    <ActionWrapper />
  </Col>
  <Col
    span={8}
    style={
      Object {
        "alignItems": "center",
        "flexWrap": "wrap",
        "justifyContent": "flex-end",
        "position": "flex",
      }
    }
  >
    <ButtonWrapper />
  </Col>
</Row>
`;
